### PR TITLE
Added new doc file to test pull request view

### PR DESCRIPTION
**Pull request by SASdemo-contributor**
# Please include answers to these questions as part of your pull request

In the GitHub webUI, use the Write tab to modify the Markdown text that is part of the pull request. For each question simply place an **X** inside the square brackets, \[X\], that represents your answer. Using the Preview tab you can see the formatted/rendered version of the message.

- **Q1: Confirm that you have the right to submit the code that is being contributed. Please consider the origin of your code and confirm you have the appropriate rights to make the submission subject to the Apache 2.0 license that applies to everything in this repository of custom steps. If so, follow the instructions for the [Contributor Agreement](ContributorAgreement.txt) (which is based on the industry-standard Developer Certificate of Origin (DCO)).**
   - [x] Yes, I have the right to submit the contributed code on behalf of myself, my company, or any other owner of the code. I have also attached my signed copy of the DCO to this message.
   - [ ] No
- **Q2: Confirm that your contribution does not include any personally identifiable information (PII), for example, in any examples used in your README file.**
   - [x] My contribution does NOT include PII data
   - [ ] My contribution includes PII data
- **Q3: Confirm your contribution does not include any encryption or other export-controlled technology.**
   - [x] My contribution does NOT contain encryption or other export-controlled technology
   - [ ] My contribution includes encryption or other export-controlled technology

### CODE REVIEW

File: docs/testdoc2.md
Comment (SASdemo-maintainer2) [inline docs/testdoc2.md:2]: Comment from maintainer as part of the pull request view. In the list of changed files, clicked inside this testdoc2.md file on a specific line.  

Here you can write line/file specific feedback for the contributor, like: "add a version history here"

Then clicked on the "Start a review" button

**Review by SASdemo-maintainer2 — CHANGES_REQUESTED** [review]
Now in the top right of the pull request there is a Submit review/ Finish your review button with a pull down menu. This pull down opens a comment text box with three options:

- Comment (submit general feedback without explicit approval)
- Approve (submit feedback and approve merging these changes)
- Request changes (submit feedback that must be addressed before merging).

I choose "Request changes)

File: docs/testdoc2.md
Comment (SASdemo-contributor) [reply docs/testdoc2.md:2]: contributor: Fake action: I didn't update the file, just added this comment and will then hit the "Resolve conversation" button in the Conversation thread of the pull request. Let's see what happens....

**Comment by SASdemo-maintainer2** [comment]
As maintainer I now see the "Merge pull request" button. Won't use it, will first check if I can do another review that requires a change to double-check whether the "Merge pull request" button is hidden when the pull request is in "Required change" mode.

**Review by SASdemo-maintainer2 — CHANGES_REQUESTED** [review]
Another request change: Rename the file to testdoc42.md to adhere to naming standards please :-)

**Comment by SASdemo-maintainer2** [comment]
Although I did another review with "Request change", the Merge pull request button is visible. There is a red circle though above it mentioning that there is a requested change. Have included a screenshot of that part in this comment. 

Still I'm going to push the Merge pull request button, let's see what happens ...

image

**Comment by SASdemo-maintainer2** [comment]
A comment from the maintainer: Added this comment and will then try to click the "merge pull request" button without the contributor having resolved the requested changes. Let's see what happens ...

Screenshot GitHub showing Changes requested and Merge pull request button

**Comment by SASdemo-contributor** [comment]
Can't see the details of the second request change, while everything worked fine for the initial requested change. Is that because the maintainer is not allowed to change their mind, and once they have finished their review with requested changes, and the contributor having resolved these requests, a maintainer cannot go into review mode again? (might be confusing myself here, might have to create a new separate pull request to test whether merge pull request is hidden/grayed when the initial review has set requested change and those changes have not been resolved yet.

**Pull request merged**
